features/step_definitions/smokey_steps.rb

```diff
@@ -57,6 +57,28 @@
   visit_path path_or_url
 end
 
+When /^I visit the "([^"]+)" finder with keywords (.*)$/ do |finder, keywords|
+  path = "/search/#{finder}?keywords=#{keywords}"
+  visit_path path
+end
+
+And /^There should be no alert$/ do
+  expect {
+    page.driver.browser.switch_to.alert.accept
+  }.to raise_error(Selenium::WebDriver::Error::NoSuchAlertError)
+end
+
+And /^I should see the string (.+)$/ do |content|
+  expect(page.find_field('keywords').value).to eq(content)
+end
+
+And /^I fill in the keyword field with (.+)$/ do |content|
+  page.fill_in 'keywords', with: content
+  page.find_field('keywords').native.send_keys(:return)
+  page.find('button[data-name="keywords"]', match: :first)
```

Comment:
Does this submit the keywords?

Reply:
line 77 does. This just waits for the tag to appear so we know the page has loaded

Reply:
Is it possible to do something like this? Like in features/step_definitions/filtering_steps.rb

```
And(/^I fill in some keywords$/) do
  fill_in 'Search', with: "Keyword1 Keyword2\n"
end
```

Reply:
fixed

Reply:
Does `"#{content}"` escape the text before it's added?

Reply:
As far as I can tell, this will emulate a human typing in that text.  So the value of the field will be the unescaped text.